integration_tests/mockApis/referrals.ts

```diff
@@ -4,6 +4,13 @@ import { apiPaths } from '../../server/paths'
 import { stubFor } from '../../wiremock'
 import type { Referral } from '@accredited-programmes/models'
 
+interface ReferralAndScenarioOptions {
+  referral: Referral
+  newScenarioState?: string
```

Comment:
Shall we make these required fields, or is there a reason behind making them nullable?

Reply:
Good point, I think `scenarioName` will always be required so will change that but you don't always need a `newScenarioState` or `requiredScenarioState`